server/seed.js

```diff
@@ -51,12 +63,18 @@ function createItems(cb) {
   ], cb);
 }
 
-function createOrders(items, cb) {
+function createEntries(items, cb) {
+  async.parallel(items.concat(items).map((item) => {
+    return async.apply(createEntry, item);
+  }), cb);
+}
+
+function createOrders(entries, cb) {
```

Comment:
The changes in this file probably aren't ideal.. but they appear to be working.. curious to see if you have any other ideas.